config/constants.ts

```diff
@@ -0,0 +1,48 @@
+import { OrcaBusStatefulConfig } from '../lib/workload/orcabus-stateful-stack';
+import { AuroraPostgresEngineVersion } from 'aws-cdk-lib/aws-rds';
+import { OrcaBusStatelessConfig } from '../lib/workload/orcabus-stateless-stack';
+
+const regName: string = 'OrcaBusSchemaRegistry';
+
+export const orcaBusStatefulConfig: OrcaBusStatefulConfig = {
+  schemaRegistryProps: {
+    registryName: regName,
+    description: 'Registry for OrcaBus Events',
+  },
+  eventBusProps: {
+    eventBusName: 'OrcaBusMain',
```

Comment:
I guess that will be used at some point in the stateless config, so probably good to pull out into global const?

Reply:
Hm... good question... 
I thought about high level constant, along the same line as
```const regName: string = 'OrcaBusSchemaRegistry';```

It is true that the value is available from the stateful stack config....
I thought with a separate high level constant the stateless stack could be independent of the stateful stack or its props structure, but the two usually go hand in hand so a bit of dependency is expected/acceptable I guess. 
Can always change down the road....

Reply:
Ah, like `regName` var. Right. I know what you mean. Ok, I will update that to realign them.